package.json

```diff
@@ -119,13 +119,13 @@
     "handlebars": "^4.7.7",
     "handlebars-loader": "^1.7.0",
     "less": "^4.0.0",
-    "less-loader": "^11.0.0",
+    "less-loader": "^11.0.0 || ^12.0.0",
     "postcss": "^8.3.0",
-    "postcss-loader": "^7.0.0",
+    "postcss-loader": "^7.0.0 || ^8.0.0",
     "sass": "^1.17.0",
     "sass-loader": "^13.0.0 || ^14.0.0",
-    "stylus": "^0.58.1",
-    "stylus-loader": "^7.0.0",
+    "stylus": "^0.60.0 || ^0.62.0",
```

Comment:
reducing the range of a peer dependency is a BC break. So you should not remove support for version 0.58